src/Plaid/Entity/ErrorCode.cs

```diff
@@ -383,6 +383,24 @@ public enum ErrorCode
 	/// <remarks><see href="https://plaid.com/docs/errors/invalid-input/#invalid_credential_fields"/></remarks>
 	[EnumMember(Value = "INVALID_CREDENTIAL_FIELDS")]
 	InvalidCredentialFields,
+
+	/// <summary>
+	/// The provided asset report token is in an invalid format. expected format: 
+	/// assets-\u003cenvironment\u003e-\u003cidentifier\u003e
+	/// </summary>
+	/// <remarks>This value is not documented on the plaid docs (as of 5/14/2022)</remarks>
```

Comment:
ISO format for world-wide consistency.

```suggestion
	/// <remarks>This value is not documented on the plaid docs (as of 2022-05-14)</remarks>
```